smartsim/_core/launcher/dragon/dragonBackend.py

```diff
@@ -452,7 +452,7 @@ def create_run_policy(
                 if run_request.policy.gpu_affinity:
                     affinity = dragon_policy.Policy.Affinity.SPECIFIC
                     gpu_affinity = run_request.policy.gpu_affinity
-
+            logger.debug(f"Affinity: {affinity}, {cpu_affinity}, {gpu_affinity}")
```

Comment:
If we are going to leave this in, can we clarify the message so we don't rely on remembering the order of the debug output:

```python
logger.debug(f"Affinity: {affinity}")
logger.debug(f"CPU affinity: {cpu_affinity}")
logger.debug(f"GPU affinity: {gpu_affinity}")
```

or do the equivalent in a single line

Reply:
Sure, let me do this!

Reply:
Per meeting w/dragon team, the affinity parameter is also being dropped and is not used. I should removed affinity and leave only cpu/cpu affinity lists behind. I think that will clear up your ordering comment?

Reply:
Agreed they are dropping the ``affinity`` parameter.  We can still specify CPU and GPU affinity at the same time (and probably want to know which is which) right?

Reply:
Yeah, we'll update it as soon as the new version is out.